Mage/src/main/java/mage/game/permanent/token/TokenImpl.java

```diff
@@ -376,11 +384,17 @@ private static void putOntoBattlefieldHelper(CreateTokenEvent event, Game game,
 
     @Override
     public void setPower(int power) {
+        if (this.backFace != null) {
+            this.backFace.setPower(power);
```

Comment:
Can you explain that commit? Why you need to modify a backface side too?

Reply:
for cards that create a token copy with modified attributes like [[croaking counterpart]] you need to apply to both faces

Reply:
Nope, characteristics modifying effects applies to active side only -- they don't see another side, so can't modify it. Cards like Croaking Counterpart must create DFC token with:
- one modified side (frog side for Croaking Counterpart);
- one normal side.

Reply:
are you sure about that? I checked with the [judge chat](https://chat.magicjudges.org/mtgrules/) and they told me it affects both sides

Reply:
Can you post the answer? There is a possibility that they were talking about the other side (if the original permanent was transformed then you must modify transformed/second side).

I'm talking about another use case:
- copy non-transformed DFC and apply new characteristics to it;
- if you transform it few turns later then you must get original side, not modified.

Reply:
the answer was just what I said there, but in both cases my understanding is that both faces are affected

Reply:
I have confirmation that it affects both sides on mtgo and arena